tests/models/pop2piano/test_modeling_pop2piano.py

```diff
@@ -507,7 +507,7 @@ def prepare_config_and_inputs_for_common(self):
 @require_torch
 class Pop2PianoModelTest(ModelTesterMixin, GenerationTesterMixin, PipelineTesterMixin, unittest.TestCase):
     all_model_classes = (Pop2PianoForConditionalGeneration,) if is_torch_available() else ()
-    all_generative_model_classes = ()
+    all_generative_model_classes = ()  # custom `generate` method
```

Comment:
might be good to explain why we can't test generate when a `custom` generate method is implemented. 
(say for example, the test will fail due to ....whatever description...)